test/functional/cursor.test.js

```diff
@@ -2346,7 +2351,7 @@ describe('Cursor', function() {
                       test.equal(null, err);
 
                       if (id === 99) {
-                        setTimeout(() => client.close());
+                        setTimeout(() => client.close(validator));
```

Comment:
This looks like a bug to me. `validator` is incrementing the close count and is meant to do so in response to the `stream` closing, not the `client`. This is short circuiting that, and likely ending the test before confirming that the `stream` has indeed closed (or errored)

Reply:
I made this change because the [test was timing out](https://evergreen.mongodb.com/test_log/mongo_node_driver_next_windows_64_vs2015_erbium_test_3.4_replica_set_unified_patch_9df093c1d46e1f8616c7a979324923205ac3dcd2_5fc7cdcf57e85a61c2ab7934_20_12_02_17_24_49/0/Cursor.should_close_dead_tailable_cursors); IIRC I tried passing `done` in instead of `validator`, but then was getting "done called twice" errors. I can take another look at this one.